repository/src/main/java/uk/gov/hmcts/ccd/definition/store/repository/model/OrganisationProfileIds.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.hmcts.ccd.definition.store.repository.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+
+import java.util.List;
+
+@ApiModel(description = "")
+public class OrganisationProfileIds {
+
+    private List<String> organisationProfileIds = null;
+
+    @ApiModelProperty(value = "")
+    @JsonProperty("organisation_profile_ids")
```

Comment:
should this be camel case?

Reply:
I am not sure when this was created so I don't know if this is documented somewhere. I will have to ask Helen.